Dockerfile

```diff
@@ -1,7 +1,24 @@
 # API Dockerfile
-
 FROM python:3.9-alpine
+
+# Copy requirements file and install them
 COPY requirements.txt .
 RUN pip3 install -r requirements.txt
-COPY src .
+
+# Copy module files
+COPY src/gateio_new_coins_announcements_bot ./src/gateio_new_coins_announcements_bot
+COPY pyproject.toml .
+COPY README.md .
+COPY setup.cfg .
+COPY setup.py .
+COPY main.py .
+
+# Install files in src/gateio_new_coins_announcements_bot as module
+RUN pip3 install -e .
+
+# Copy relevant files to run bot
+COPY config.yml .
+COPY old_coins.json .
+COPY auth/auth.yml ./auth/
```

Comment:
- [ ] Are these all files? Or did I miss any?

Reply:
Yes they are
- config is needed for the configuration setup
- old_coins is needed to exlcude bought/sold coins
- auth is needed to grab the API keys

Reply:
I'd also suggest moving all the configs (config.yml, auth.yml) in a single `config` dir

Reply:
I like that idea, but it should be done in another issue/PR because it requires additional changes which are unrelated to this one.